avromatic.gemspec

```diff
@@ -20,11 +20,13 @@ Gem::Specification.new do |spec|
   spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ['lib']
 
-  spec.required_ruby_version = '>= 2.5'
+  spec.metadata['rubygems_mfa_required'] = 'true'
 
-  spec.add_runtime_dependency 'activemodel', '>= 5.2', '< 6.2'
-  spec.add_runtime_dependency 'activesupport', '>= 5.2', '< 6.2'
-  spec.add_runtime_dependency 'avro', '>= 1.9.0', '< 1.11'
+  spec.required_ruby_version = '>= 2.7'
+
+  spec.add_runtime_dependency 'activemodel', '>= 5.2', '< 7.1'
+  spec.add_runtime_dependency 'activesupport', '>= 5.2', '< 7.1'
+  spec.add_runtime_dependency 'avro', '>= 1.9.0', '< 1.12'
```

Comment:
How would you feel about dropping Avro 1.9 support since we're already doing a major version bump?

Reply:
No objection since it is already almost two years old. I've added a commit that does that. I can squash the commits before merging.

At the same time, I added testing against Ruby 3.1 since there is now a compatible Rails 7.0 release.

I also excluded the `gemfiles/` directory for Rubocop so that flagging offenses and auto-correcting those generated files.